docs/how-to/kafka-ui.md

```diff
@@ -0,0 +1,144 @@
+(how-to-kafka-ui)=
+# How to facilitate administrative tasks using Kafbat's Kafka UI
+
+Although administration of a Charmed Apache Kafka cluster is possible using the command line interface (CLI) and various utilities shipped with the Apache Kafka snap, some administrators prefer a graphical user interface (GUI) which facilitates monitoring of the Apache Kafka cluster and performing various administrative tasks. Therefore, charmed Apache Kafka solution includes a charmed operator for [Kafbat's Kafka UI](https://github.com/kafbat/kafka-ui), which enables users to:
+
+- View Apache Kafka cluster configuration, topics, ACLs, consumer groups and more
+- Broker performance monitoring via JMX metrics dashboards
+- Seamless integration with other Charmed Apache Kafka operators, like [Charmed Apache Kafka Connect](https://charmhub.io/kafka-connect) and [Charmed Karapace](https://charmhub.io/karapace)
+
+In this guide, you will deploy the Charmed Kafka UI operator, integrate it with the Charmed Apache Kafka and other related products, and enable authentication and TLS for securing access to the Kafka UI.
```

Comment:
Suggestion: Let's try to make it a bit more dry, as this is a how-to guide and not a tutorial.

```suggestion
In this guide, you will:

- Deploy the Charmed Kafka UI operator
- Connect it to Charmed Apache Kafka and related products
- Configure authentication and TLS to secure access to Kafka UI
```